src/ApiFetch.js

```diff
@@ -0,0 +1,23 @@
+class ApiFetch {
+  constructor() {
+    this.root = 'https://fe-apps.herokuapp.com/api/v1/travel-tracker/data'
+  }
+
+  getTravelersData() {
+    let url = `${this.root}/travelers/travelers`
```

Comment:
Never thought about turning the ApiFetch into a class. This is really clean and makes it very easy to read and see what is happening!